docs/dev/development.adoc

```diff
@@ -180,13 +180,20 @@ NOTE: Refer https://github.com/golang/go/wiki/LearnTesting for Go best practices
 
 *Prerequisites:*
 
-* A `minishift` or OpenShift environment with Service Catalog enabled:
+* A `minishift` or OpenShift 3.11 environment with Service Catalog enabled:
 +
 ----
 $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker,template-service-broker"
 ----
+OR
+* A `crc` environment for 4.* local cluster:
++
+For `crc` installation please refer https://github.com/code-ready/crc#documentation
```

Comment:
I think making this short will be preferable. Like instead of `For crc installation please refer https://github.com/code-ready/crc#documentation` you can simply use `Install https://github.com/code-ready/crc#documentation[crc]

Reply:
How the phrase ```Follow link:https://github.com/code-ready/crc#documentation[crc] installation guide.``` is ?